src/main/webapp/i18n/de/overview.json

```
@@ -4,7 +4,8 @@
             "teams": {
                 "title": "Teams",
                 "preExpiresIn": "Gültigkeit läuft in",
-                "postExpiresIn": "Tagen ab"
+                "postExpiresIn": "Tagen ab",
+                "official": "offiziell"
```

According to the ticket this should be:
```suggestion
                "official": "Projektteam"
```